### PR TITLE
Adds blob toxin to blob description

### PR DESCRIPTION
**Pull request by AnturK**
What it says on the tin. Playing guess the color gets annoying after the first time.

"It looks like it's of a [toxin name here] kind."  on examine

### CODE REVIEW

**Comment by Jalleo** [comment]
That is too easy. A better option may be making the currebt tools that can scan reagents in people be able to scan the blob to see its reagent.

**Comment by Iamgoofball** [comment]
No, this is good, Jalleo. The point of the colors was for rapidfire "OH SHIT ITS X BLOB" but that didn't work so this is better.

File: code/game/gamemodes/blob/theblob.dm [hunk 1]
```diff
@@ -196,6 +198,11 @@
 		color = a_color
 	return
 
+/obj/effect/blob/proc/update_desc(var/a_desc=null)
```
Comment (Cheridan) [inline code/game/gamemodes/blob/theblob.dm]: Is all this even necessary? Can't it just read the reagent off in the examine proc?

Comment (AnturK) [inline code/game/gamemodes/blob/theblob.dm]: Not without changing how blob works a bit.  Blob parts(except the core) dont track their origin/ mind/reagent at all so this needs to be updated when it changes type/other blob affects it/chem rerolls

Comment (AnturK) [inline code/game/gamemodes/blob/theblob.dm]: I guess i could make it poll for overmind with the same color instead

Comment (Cheridan) [inline code/game/gamemodes/blob/theblob.dm]: I think that would be more sane. Just set the desc whenever the blob part reagent is first added. Then it'll just be in one place.

**Comment by Razharas** [comment]
Merge conflicts

**Comment by phil235** [comment]
beep boop, still working on that @AnturK ?

**Comment by AnturK** [comment]
Oh totally forgot about this one. Will fix the conflicts later today.

**Comment by Razharas** [comment]
>Will fix the conflicts later today.

Famous last words

**Comment by AnturK** [comment]
Don't believe the timestamps. This was here 3 days ago. Honest.

**Comment by Razharas** [comment]
And what about cheris comment?

**Comment by AnturK** [comment]
Now it polls by color. No idea if every blob has unique color though.

File: code/game/gamemodes/blob/theblob.dm [hunk 2]
```diff
@@ -187,6 +188,7 @@
 	var/obj/effect/blob/B = new type(src.loc)
 	if(!istype(type, /obj/effect/blob/core) || !istype(type, /obj/effect/blob/node))
 		B.color = color
+		B.desc = desc
```
Comment (phil235) [inline code/game/gamemodes/blob/theblob.dm]: Is this necessary now? Afaik all obj/effect/blob have the same desc var.

Comment (AnturK) [inline code/game/gamemodes/blob/theblob.dm]: Oh right. Forgot to remove this.

**Comment by AnturK** [comment]
Should be good now.

**Pull request merged**